### PR TITLE
[DML] Resize-18 registration

### PR DESCRIPTION
**Pull request by linnealovespie**
### Description

Register resize-18, which will be lit up automatically when dml feature level bumps up to 6300.

### CODE REVIEW

File: onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/OperatorRegistration.cpp [hunk 1]
```diff
@@ -1068,7 +1068,6 @@ constexpr static OperatorRegistrationInformation operatorRegistrationInformation
     {REG_INFO_MS(   1,  MultiHeadAttention,                 typeNameListAttention,          supportedTypeListAttention,             DmlGraphSupport::Supported)},
     {REG_INFO_MS(   1,  QLinearConcat,                      typeNameListQLinearConcat,      supportedTypeListQLinearConcat,         DmlGraphSupport::Supported)},
     {REG_INFO_MS(   1,  RotaryEmbedding,                    typeNameListRotaryEmbedding,    supportedTypeListRotaryEmbedding,       DmlGraphSupport::Supported)},
-    {REG_INFO_MS(   1,  QLinearConcat,                      typeNameListQLinearConcat,      supportedTypeListQLinearConcat,         DmlGraphSupport::Supported)},
```
Comment (fdwr) [inline onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/OperatorRegistration.cpp:1043]: Why is QLinearConcat deleted? That seems unrelated. 🤔

Comment (linnealovespie) [reply onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/OperatorRegistration.cpp:1043]: It's a repeat of line 1069, removing a typo.

Comment (fdwr) [reply onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/OperatorRegistration.cpp:1043]: Gotcha. Look at your files tab though, as there's a lot of other noise too:

image
